.gotestfmt/github/package.gotpl

```diff
@@ -24,11 +24,11 @@ we are creating a stylized header for each package.
             {{- if or (not $settings.HideSuccessfulTests) (ne .Result "PASS") -}}
                 ::group::
                 {{- if eq .Result "PASS" -}}
-                    {{ "\033" }}[0;32m✅
+                    {{ "\033" }}[0;32m✅{{if $settings.ShowTestStatus}} ({{.Result}}){{end}}
```

Comment:
The explicit test status would be better placed next to the duration below, something like:

```
✅ NameHere (PASS; 31s)
```

Reply:
Addressed, please take another look.